src/Session.ts

```diff
@@ -13,11 +14,15 @@ export default class Session {
   context: Context | null
   store: Store
   expiration: number | null
+  cookieSetOption: CookiesSetDeleteOptions
+  cookieGetOption: CookiesGetOptions
 
-  constructor (store : Store = new MemoryStore, options? : SessionOptions) {
+  constructor (store : Store = new MemoryStore, options? : SessionOptions, cookieSetOption? : CookiesSetDeleteOptions, cookieGetOption? : CookiesGetOptions) {
```

Comment:
It might be better to put this within the existing `SessionOptions` interface, so we don't make the number of constructor arguments huge.

So the end experience is like

```ts
const session = new Session(store, {
  expireAfterSeconds: 900,
  cookieSetOptions: {}
  cookieGetOptions: {}
} 
```

Also, the keys should be plural to indicate multiple options, `cookieSetOptions` instead of `cookieSetOption`, same for get options

Reply:
Yeah, it's a better idea. I made it in the latest commit.